tests/pinkopy/test_base_session.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+
+from pinkopy.base_session import BaseSession
+from tests.pinkopy import test_helper
+
+
+class TestPinkopyError(unittest.TestCase):
+    pass
+
+
+class TestBaseSessionMethods(unittest.TestCase):
+    def test__init__(self):
+        expected = test_helper.mock_session(BaseSession)
+        base_session = expected['Session']
+        test_helper.validate_base_session(expected, base_session)
+
+    def test__enable_method_cache(self):
+        base_session = test_helper.mock_session(BaseSession)['Session']
+        for method_name in base_session.cache_methods:
+            method = getattr(base_session, method_name)
+            assert method.ttl_cache() is not None
```

Comment:
I can't figure out what is being tested here. How would this method get the function `ttl_cache()` on it?